content/participants/ttjc@ravisaxena23/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Ravi Nandan Saxena
+type: "participant"
+name: "Ravi Nandan Saxena"
+portfolio: "http://bit.ly/ravisresume"
```

Comment:
Portfolio should contain the link to your own personal website not your resume. If you dont have one right now leave it blank and update it later.